Mathlib/MeasureTheory/Function/L1Space.lean

```diff
@@ -446,8 +444,6 @@ theorem integrable_def {α} {_ : MeasurableSpace α} (f : α → β) (μ : Measu
     Integrable f μ ↔ (AEStronglyMeasurable f μ ∧ HasFiniteIntegral f μ) :=
   Iff.rfl
 
-attribute [eqns integrable_def] Integrable
```

Comment:
This one doesn't have a porting note, but I strongly suspect it was another case of leanprover/lean4#2243 and so the lemma can be deleted too.